jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java

```diff
@@ -40,113 +38,117 @@
  */
 public class ErrorManageUtil {
 
-    /** The log. */
+    /**
+     * The log.
+     */
```

Comment:
Thanks for applying basic style.
Let's get rid of these useless comments - waste of visual space and IDE and distraction.

Reply:
Same for all similar basic comments below.